cmd/gen-go-sample/main.go

```diff
@@ -505,7 +511,7 @@ func (g *generator) handleOut(meth *descriptor.MethodDescriptorProto, valSet Sam
 // prependLines adds prefix to every line in b. A line is defined as a possibly empty run
 // of non-newlines terminated by a newline character.
 // If b doesn't end with a newline, prependLines panics.
-func prependLines(b *bytes.Buffer, prefix string) {
+func prependLines(b *bytes.Buffer, prefix string, skipEmptyLine bool) {
```

Comment:
Please comment on the semantics of `skipEmptyLine`. I take it that if true, it does not add the prefix to an empty line, and if false, it does?